project/src/main/webapp/script.js

```diff
@@ -134,28 +134,40 @@ function createPlaceElement(place) {
   const name = document.createElement('li');
   name.innerText = place.name;
   placeElement.appendChild(name);
-  placeElement.appendChild(document.createElement('br'));
   // Add link to website
   if (place.websiteUrl) {
-    const websiteLink = document.createElement('a');
-    websiteLink.href = place.websiteUrl;
-    websiteLink.title = place.websiteUrl;
-    websiteLink.innerText = 'Restaurant\'s website';
-    placeElement.appendChild(websiteLink);
-  } else {
-    const noSite = document.createTextNode('We don\'t have a link to the restaurant\'s website.');
-    placeElement.appendChild(noSite);
+    addLinkToPlaceElement(placeElement, place.websiteUrl, 'Website');
+  }
+  if (place.googleUrl) {
+    addLinkToPlaceElement(placeElement, place.googleUrl, 'Google Maps link');
```

Comment:
I think 'Google Maps link' is a confusing description (since this doesn't lead to google maps). We can use the description of this field fromm the Places API: "Google page link"

Reply:
Interesting point, thought it does lead to Google Maps. Did you try to run this and press the link? I think this is a correct description

Reply:
Yeah it opens the map with the place's google page on the side. I now see the address is google maps so technically you are right, but I think when the user will see this description he won't understand this is to get more information about the place (such as opening hours, reviews and more info that appears on this link). I don't have strong feelings about it so we can leave like this if you prefer haha. And maybe one day we pilot and get actual users feedback about it :)

Reply:
That's a great idea! You are right, maybe it's clear for me but not for everyone.
I'll leave it for now